hatchery/nextflow.go

```diff
@@ -964,6 +963,32 @@ func createS3bucket(s3Svc *s3.S3, kmsSvc *kms.KMS, bucketName string, kmsTags []
 		return "", err
 	}
 
+	expirationDays := nextflowGlobalConfig.S3ObjectsExpirationDays
+	if expirationDays <= 0 {
```

Comment:
Will there be any case where the expirationDays value is less than zero?

Reply:
it's 0 if unspecified and it could be anything if someone configures it wrong